node/test_force.js

```diff
@@ -286,6 +287,16 @@ function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
 }
 
 //
+// Update createApp.sh to not do pod install
+// 
+function editCreateAppToNotDoPodInstall(tmpDir) {
+    log('Editing  createApp.sh to not do pod install', COLOR.green);
+    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', 'forceios', 'build', 'app_template_files', 'createApp.sh'), new RegExp('pod install$', 'g'), '# pod install');
```

Comment:
Strictly speaking, wouldn't a more bullet-proof regexp be:

`new RegExp('^\s*pod install', 'g')`

?